src/analytics/analytics.ts

```diff
@@ -136,3 +136,33 @@ export interface SubscriberMetrics {
   total_subscribers: number
   date: string
 }
+
+export interface RevenueMetricsValue {
+  label: string
+  distinct_person_count: number
+  conversation_marketing_count: number
+  conversation_utility_count: number
+  journeys_count: number
+  journeys_revenue_gross_total: number
+  journeys_revenue_net_total: number
+  campaigns_count: number
+  campaigns_revenue_net_total: number
+  campaigns_revenue_gross_total: number
+  attribution_event_count: number
+  per_conversation_mean: number
+  revenue_gross_total: number
+  revenue_net_total: number
+  timeframe_start: string
+  timeframe_end: string
```

Comment:
todo: Lets leave a comment for the that the string is in fact DateTime ISO string